site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,390 @@
++++
+date = "2023-02-10T18:15:00+00:00"
+title = "Citra Mega Progress Report 2020 Q2~2023 Q1"
+tags = [ "progress-report" ]
+author = "autumnburra"
+coauthor = "flTobi"
+coauthor2 = "sleepingsnake"
++++
+
+Welcome back to another Citra Mega Progress Report!
+
+Yes! We aren't dead! First off, we do apologize for the wait, we are still lacking a full time writer. 
+In the meantime, Citra community moderator [autumnburra](https://community.citra-emu.org/u/autumnburra/summary) and developer [FearlessTobi](https://community.citra-emu.org/u/fltobi/summary) have come together with the assistance of another community moderator, [SleepingSnake](https://community.citra-emu.org/u/sleepingsnake/summary), to provide you, the Citra community, with this awesome report on all the changes we’ve had in Citra since 2020 Q2!
+
+If you keep up to date with messages posted in the `#development` channel in our [Discord server](https://citra-emu.org/discord/), you may already be aware of some of these! But nevertheless, this is a read that you’re not gonna want to miss out on! 
+
+# Contents
+
+1. [New Features]({{< relref "#new-features" >}})
+1. [Android]({{< relref "#android" >}})
+1. [Emulation Accuracy]({{< relref "#emulation-accuracy" >}})
+1. [Frontend Improvements]({{< relref "#frontend-improvements" >}})
+1. [Surprise Annoucement]({{<relref "#surprise-annoucement" >}})
+1. [Conclusion]({{<relref "#conclusion" >}})
+
+# New Features
+
+## Implement basic rerecording features ([#5448](https://github.com/citra-emu/citra/pull/5448)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+Following on from the implementation of a built-in video dumper, mentioned in our previous Progress Report, is the addition of more features to aid all of your video recording and TAS needs!
+
+Being able to use rerecording in an emulator is key when wanting to create TAS videos! These would include features such as adjusting the emulation speed and being able to sync-robust save states. Sync-robustness is the concept that if a save state is launched a certain number of times, it should behave the exact same way each time.
+
```

Comment:
I recommend including how to reach new options, step-by-step. And/or include a picture of the location in Citra's settings.

Reply:
Agree, although since it's a big report, it might be fine to just include a one-liner "you can find X under the Y->Z menu"